CODEOWNERS

```diff
@@ -1 +1 @@
-* @mickmister @spirosoik
+* @wiggin77 @spirosoik
```

Comment:
@spirosoik Do you want to continue to be listed as a Codeowner?

Reply:
@hanzei I think so https://github.com/mattermost/mattermost-plugin-gitlab/pull/302#discussion_r944278711